### DISCUSSION TITLE
How to define noscript?

### DISCUSSION THREAD

**Discussion by ivanjaros** [quasarframework/quasar#17320] [General - Components / Directives / etc] [answered]
I was trying to add noscript content to my SPA. First, via meta plugin, which I already use, then directly editing the index file. Neither approached worked, which is quite strange.

What is the recommended way then?

**Comment by yusufkandemir** [comment, marked as answer]
Adding it to `index.html` should work fine. Please share the way you did it and the way you verified that it didn't work.

Example:
```html
<!doctype html>
<html>
  <head>
    <title><%= productName %></title>
    ...
  </head>

  <body>
     <noscript>
       <strong>Sorry, but <%= productName %> doesn't work without JavaScript.</strong>
       Please enable JavaScript to continue.
     </noscript>

     <!-- quasar:entry-point -->
  </body>
</html>
```

**Reply by ivanjaros** [reply]
It works in body, does not work in head(which is valid). I tried it before in body as well but it did not work, not sure why it does now...who knows.

**Reply by yusufkandemir** [reply]
It's valid in `<head>`, but for cases like loading a specific CSS file when JS is disabled. You can't display content in `<head>`.
